hooks/pre_start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OPEN_FILES_RECOMMENDED=1024
```

Comment:
I think this is too low. We mine a block/~5min, each kept in separate file (sure, we close FDs, but what if we restart and re-read?). Also each outgoing connection to a peer takes an FD. Can we make it 1024*256?

Reply:
You mean 262144?

Default hard limit on linux (ubuntu 16) is 65536. Raising hard limit is more sophisticated for users and they'd have hard time raising it.

```
vagrant@epoch:/app$ ulimit -Hn
65536
```

And this is OSX:
```
$ sysctl -a | grep files
kern.maxfiles: 49152
kern.maxfilesperproc: 24576
kern.num_files: 4633
```

Reply:
So, 24576 is the max portable (for our supported operating systems: Ubuntu & OSX).

Reply:
Agreed on slack that 24576 is fine for average user